admin_manual/configuration_files/encryption_configuration.rst

```diff
@@ -122,6 +124,8 @@ These are:
 .. note::
    Please note, the commands have to be run in this order.
 
+.. warning:: If you already have Basic encryption enabled - you **must not** enable Master Key Based encryption.
```

Comment:
I had no idea about encryption and tried to enable it following the docu, enabled the default encryption and then enabled master key encr and got errors when I created files.